spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/ComposedRunnerJobFactory.java

```diff
@@ -109,7 +109,9 @@ public Job getObject() throws Exception {
 						.start(createFlow())
 						.end())
 				.end();
-		builder.incrementer(new UuidIncrementer());
+		if(this.composedTaskProperties.isUuidInstanceEnabled()) {
```

Comment:
**[TinyNit]** I am not sure if our formatter needs updating, but I am suprised it is not balking about `space after if`.

Reply:
Will polish.  Thanks for noticing!